src/System.Xml.XmlSerializer/src/System.Xml.XmlSerializer.csproj

```diff
@@ -11,9 +11,12 @@
     <DebugSymbols>true</DebugSymbols>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <ProjectGuid>{D62A6082-5229-4845-8BE9-75753E08C65A}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netcore50'">$(DefineConstants);NET_NATIVE</DefineConstants>
```

Comment:
I see that the internal build also has `<EnableWinRT>true</EnableWinRT>` in the project file. Do we need that here?